src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -22,9 +27,16 @@ public class FindCommand extends Command {
     public static final String MESSAGE_PURPOSE = "Finds all employees and/or potential hires whose names "
             + "contain any of the specified keywords (case-insensitive) and displays them as a list with "
             + "index numbers.";
-    public static final String MESSAGE_FORMAT = COMMAND_WORD + " (" + FindEmployeeCommand.ARGUMENT_WORD
-        + "/" + FindPotentialCommand.ARGUMENT_WORD + "/" + FindCommand.ARGUMENT_WORD + ") KEYWORD [MORE_KEYWORDS]";
-    public static final String MESSAGE_EXAMPLE = COMMAND_WORD + " " + ARGUMENT_WORD + " n/alice bob e/alice@gmail.com";
+    public static final String COMMAND_PARAMETERS = FindEmployeeCommand.ARGUMENT_WORD + " for employees, "
+            + FindPotentialCommand.ARGUMENT_WORD + " for potential hires, and "
+            + FindCommand.ARGUMENT_WORD + " for both employees and potential hires.";
+    public static final String COMMAND_KEYWORDS = PREFIX_NAME + "/[NAME] " + PREFIX_PHONE + "/[PHONE NUMBERS] "
```

Comment:
Should NAME be NAMES instead?
Also can you use PHONE_NUMBERS instead as it follows the same format as edit?